homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -265,17 +264,34 @@ def _arming_time(self, state):
         """Get the arming time."""
         return self._arming_time_by_state[state]
 
-    def _pending_time(self, state):
+    def _delay_time(self, state):
```

Comment:
I'm not 100% sure this function needs to change names, but if you do you should also change the function description.  Note that "pending time" is the time from when we receive the request to trigger to when it actually transitions to the triggered state.  I.e., it goes from ARMED -> PENDING -> TRIGGERED and sits in the pending state for pending_time.   On the other hand, for some reason this is called `delay_time` in the config, so yes, it is somewhat confusing.